src/lumigo_tracer/wrappers/sql/sqlalchemy_wrapper.py

```diff
@@ -40,7 +45,7 @@ def _before_cursor_execute(conn, cursor, statement, parameters, context, execute
 
 def _after_cursor_execute(conn, cursor, statement, parameters, context, executemany):
     with lumigo_safe_execute("handle sqlalchemy after execute"):
-        span = SpansContainer.get_span().get_last_span()
+        span = SpansContainer.get_span().get_span_by_id(_last_span_id)
```

Comment:
In this file, I think we still take only the last_span. I'm not sure it will always work with threads.

Reply:
yes. I preferred to not support sqlalchemy with threads. this is too niche